backend/functions/src/middleware/can-edit-profile.ts

```diff
@@ -0,0 +1,11 @@
+import { NextFunction, Request, Response } from 'express';
+
+export const canEditProfile = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    if (req.user.uid === req.params.id) {
+      next();
+    } else throw new Error("'Authorization Error'");
```

Comment:
You dont need the single quotes inside the double ones.

I think the error message should be more clear.